pkg/apparchive/helm-v1beta2.go

```diff
@@ -140,20 +165,40 @@ type HelmWriteOptions struct {
 	Clientset           kubernetes.Interface
 }
 
-// WriteRenderedHelmCharts writes the rendered helm chart to the rendered directory
+// WriteRenderedHelmCharts writes the rendered helm chart to the rendered directory and processes the images
 func WriteRenderedHelmCharts(opts HelmWriteOptions) error {
 	if opts.KotsKinds == nil || opts.KotsKinds.V1Beta2HelmCharts == nil {
 		return nil
 	}
 
 	for _, downstream := range opts.Downstreams {
 		for _, helmChart := range opts.KotsKinds.V1Beta2HelmCharts.Items {
-			renderedPath, err := renderHelmChart(opts, downstream, &helmChart)
+			// template the chart with the values to the rendered dir for the downstream
+			renderedPath := path.Join(opts.RenderedDir, downstream, "helm", helmChart.GetDirName())
+			if err := templateHelmChartWithValuesToDir(opts.HelmDir, &helmChart, "values.yaml", renderedPath, opts.Log.Debug); err != nil {
+				return errors.Wrap(err, "failed to template helm chart for rendered dir")
+			}
+
+			if !opts.ProcessImageOptions.RewriteImages || opts.ProcessImageOptions.AirgapRoot != "" {
```

Comment:
I added this back since we only want to process images for online.  Is that correct?

Reply:
right, but `|| opts.ProcessImageOptions.AirgapRoot != ""` isn't needed here because you can't have an airgap install/bundle without an on-prem registry (e.g. `RewriteImages` will always be `true` in airgap)

Reply:
oh, you're right. my bad.